src/main/java/pilot/cards/pilot/Alternator.java

```diff
@@ -22,23 +23,37 @@ public class Alternator extends CustomPilotModCard {
     private static final int DAMAGE = 2;
     private static final int UPGRADE_PLUS_DMG = 2;
 
+    private static final int DAMAGE_PER_MOMENTUM = 1;
+
     public Alternator() {
         super(ID, COST, TYPE, COLOR, RARITY, TARGET);
         baseDamage = DAMAGE;
+        magicNumber = baseMagicNumber = DAMAGE_PER_MOMENTUM;
     }
 
     @Override
     public boolean shouldGlowGold() {
         return ((Pilot)AbstractDungeon.player).hasMomentum();
     }
 
+    @Override
+    protected int calculateBonusBaseDamage() {
+        AbstractPlayer p =AbstractDungeon.player;
+        int bonusDamage = 0;
+        if (p.hasPower(MomentumPower.POWER_ID)) {
+            bonusDamage = p.getPower(MomentumPower.POWER_ID).amount * magicNumber;
+        }
+        return bonusDamage;
+    }
+
     @Override
     public void use(AbstractPlayer p, AbstractMonster m) {
-        if (((Pilot)p).hasMomentum() && p.hasPower(DexterityPower.POWER_ID)) {
-            this.baseDamage += p.getPower(DexterityPower.POWER_ID).amount;
+        if (((Pilot)p).hasMomentum() && p.hasPower(MomentumPower.POWER_ID)) {
+            this.baseDamage += p.getPower(MomentumPower.POWER_ID).amount;
         }
         this.addToBot(new AttackDamageRandomEnemyAction(this, AttackEffect.SLASH_HORIZONTAL));
         this.addToBot(new AttackDamageRandomEnemyAction(this, AttackEffect.SLASH_HORIZONTAL));
+        this.baseDamage -= p.getPower(MomentumPower.POWER_ID).amount;
```

Comment:
Sweet, yeah I wasn't quite sure what was happening here. Essentially, I noticed that the card was keeping the scaling with your momentum, meaning that if you increased it by 5 and then drew it on a later turn, it would still be increased by that 5 on top of your current momentum. I decided to just add the bit that subtracted by your current momentum. I see what I need to do here.